include/aikido/rviz/TrajectoryMarker.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef AIKIDO_RVIZ_TRAJECTORYMARKER_HPP_
+#define AIKIDO_RVIZ_TRAJECTORYMARKER_HPP_
+
+#include <dart/dynamics/Frame.hpp>
+#include <interactive_markers/interactive_marker_server.h>
+#include <visualization_msgs/InteractiveMarker.h>
+#include "aikido/trajectory/Trajectory.hpp"
+
+namespace aikido {
+namespace rviz {
+
+/// A wrapper class of RViz InteractiveMarker for visualizing AIKIDO trajectory
+/// in RViz.
+class TrajectoryMarker final
+{
+public:
+  /// Constructor.
+  /// \param[in] markerServer RViz marker server.
+  /// \param[in] frameId RViz frame ID.
+  /// \param[in] trajectory C-sapce (or joint-space) trajectory.
+  /// \param[in] skeleton Target DART meta skeleton that the C-space trajectory
+  /// will be applied to compute the visualizing task-space trajectory.
+  /// \param[in] frame Target DART frame where the trajectory of its origin
+  /// will be visualized.
+  /// \param[in] rgba Color and alpha of the visualizing task-space trajectory.
+  /// Default is [RGBA: 0.75, 0.75, 0.75, 0.75].
+  /// \param[in] thickness Thickness of the visualizing task-space trajectory.
+  /// Default is 0.01.
+  /// \param[in] numLineSegments Number of line-segments of the visualizing
```

Comment:
I think `numLineSegments` is a somewhat unintuitive way to discretize the trajectory. More comments below.